### ISSUE TITLE
Possible to get Stripe's invoice payment URL?

### ISSUE THREAD

**Issue by iamkeir** [enupal/stripe#59]
If a subscription is `past_due`, is it possible to get the Stripe checkout payment URL from available data? E.g. `https://pay.stripe.com/invoice/invst_kij40kj43oj232kl3jkj3`

I would like to be able to display this link to the user so they can go ahead and pay.

**Comment by andrelopez** [comment]
Hi @iamkeir Please follow the next example:

```
{% for order in craft.enupalStripe.getSubscriptionsByUser(currentUser.id) %}
            {% set subscription = order.getSubscription() %}
            {% if subscription.status == 'past_due' %}
                {% set invoice= subscription.data.latest_invoice %}
            {% endif %}
        {% endfor %}
```
Try to {{ dump(subscription) }} and check if you can get the info from there.

**Comment by iamkeir** [comment]
Unfortunately not - there is nothing in either `order`, `order.getSubscription` or `order.getSubscription.data` that contains any of the Stripe checkout URL data.

Could I re-open this as a feature request instead? The purpose is to allow the customer to fix billing problems themselves, by going to the Stripe invoice URL. At the moment, there is no way for customers to resolve billing issues - unless I have missed something in the docs.

**Comment by iamkeir** [comment]
Actually, perhaps relates to https://github.com/enupal/stripe/issues/25 as billing issues usually relate to a card problem, so if once this feature is built we can show a change/edit card form to the user, they would be able to resolve.

**Comment by andrelopez** [comment]
Hi @iamkeir I'm working on ticket #25 hope will solve this issue.

**Comment by andrelopez** [comment]
Hey @iamkeir #25 is released in v2.1.6
